test/test-lib/init-tests.ts

```diff
@@ -0,0 +1,54 @@
+import { app } from '../../init';
+import * as balenaToken from './balena-token';
+import * as fixtures from './fixtures';
+import { supertest, UserObjectParam } from './supertest';
+
+export const preInit = async () => {
+	await import('./aws-mock');
+
+	// override the interval used to emit the queue stats event...
+	const { DeviceOnlineStateManager } = await import(
+		'../../src/lib/device-online-state'
+	);
+	(DeviceOnlineStateManager as any)['QUEUE_STATS_INTERVAL_MSEC'] = 1000;
```

Comment:
Ugh, this is a bit ugly -- that class should get that config value from its environment and tests should override it appropriately instead of having to patch it like this in tests (though I'm not suggesting it should be changed with this PR).